README.md

```diff
@@ -42,6 +42,8 @@ They are used when adding new services or grouping services together (and do not
 | env     | `JOBS_ENABLED`     | `true`: rabbitmq,archivemock,jobs feature                                       | `''`    | v3                    | Creates a rabbitmq message broker which the be posts to and the archivemock listens to. It emulates the data long-term archive/retrieve workflow                       |                         |
 | env     | `ELASTIC_ENABLED`  | `true`: elastic,elastic feature                                                 | `''`    | v4                    | Creates an elastic search service and sets the be to use it for full-text searches                                                                                     |                         |
 | env     | `LDAP_ENABLED`     | `true`: ldap auth                                                              | `''`    | as set                | Creates an LDAP service and sets the be to use it as authentication backend                                                                                                    |                         |
+| env     | `OIDC_ENABLED`     | `true`: oidc auth                                                              | `''`    | as set                | Creates an OIDC identity provider and sets the be to use it as authentication backend                                                                                                    |                         |
```

Comment:
I think the `as set` should be `'*'` instead

Reply:
Also did it for LDAP